docs/api/sql/Function.md

```diff
@@ -1668,3 +1668,17 @@ SELECT ST_ZMin(ST_GeomFromText('LINESTRING(1 3 4, 5 6 7)'))
 
 Output: `4.0`
 
+## ST_NumPoints
```

Comment:
Docs need to be sorted alphabetically. Please explain the behavior if a non-linestring geometry is given.

Reply:
Updated documentation and corrected order